airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -253,10 +253,15 @@ def get_batch_client(self, region: str | None = None) -> BatchControllerClient:
             credentials=self.get_credentials(), client_info=CLIENT_INFO, client_options=client_options
         )
 
-    def wait_for_operation(self, operation: Operation, timeout: float | None = None):
+    def wait_for_operation(
+        self,
+        operation: Operation,
+        result_retry: Retry | _MethodDefault = DEFAULT,
+        timeout: float | None = None,
```

Comment:
Better to swap the positions of these two to avoid breaking compatibility.

Reply:
Fixed
Thanks for the review!